contents/handbook/engineering/product-design.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Product Design, for Engineers
+sidebar: Handbook
+showTitle: true
+---
+
+It is up to you when to involve our product designers in your work.
+
+## v0.1 or v2?
+
+As the feature owner, you should make a choice if you're building a very basic first iteration of something, or if you're improving the experience.
+
+If you're improving an existing feature that is popular, you are probably creating v2.
+
+If you're shipping an entirely new feature (i.e. SQL for PostHog), then you should figure out if anyone cares, which usually means creating a v0.1, and releasing it behind a feature flag to some friendly users (pro tip: make friends by being support hero).
+
+Please _communicate_ to product design what your expectations are!
+
+### T-shirt sizes
+
+The more complex a feature is to implement, the more likely it is that involving product design will make you faster.
+
+### Your design skill
+
+We generally hire full stack engineers, but some people think more like designers than others. This is fine - you should play to your strengths.
+
+The less strong you are at design, the more we'd encourage you to involve a product designer.
+
+If you're unsure about your skill level, ask a product designer for direct feedback. [This is a book we'd recommend](https://en.wikipedia.org/wiki/The_Design_of_Everyday_Things) if you want to learn the mindset.
+
+### If product design has capacity
+
+Sometimes product design may push back if they simply haven't got capacity. It's subjective when this may happen, and it'll usually be in cases where they feel they won't be as helpful based on the above.
```

Comment:
This part makes it feel like design is optional. If so, we should clearly state that's the case. 

I'd still argue for clearly making it optional but suggested for v0.1 (that you release to beta users just to get a response), but necessary for a public unflagged and marketed v2. We can also agree that there will be exceptions or emergencies where we step on toes, and agree that we will not do so by default.

If we don't have enough product design capacity to get approvals in a timely manner, let's hire more 🤷.